src/main/docker/Dockerfile

```diff
@@ -54,9 +54,6 @@ RUN chmod +x root/startup/startup.d/*
 #
 FROM cafapi/opensuse-jre11:2
 
-RUN zypper -n refresh && \
-    zypper -n update
-
```

Comment:
After consulting with Michael on failing jenkins build this was removed due to some permissions issue.  I can revert and raise its own PR if required.

Reply:
Yes remove it from this one, but feel free to check it straight into the main branch without another PR.  It doesn't need to be reviewed.